lib/Pipeline.js

```diff
@@ -85,6 +85,7 @@ Pipeline.processJSON = function(gltf, options) {
  * @param {Object} [options.quantize] Flag to run quantizeAttributes stage.
  * @param {Object} [options.preserve=false] Flag to turn optimization pipeline stages on/off to preserve the original glTF hierarchy.
  * @param {Boolean} [options.mergeVertices=false] Flag to merge duplicate vertices, which can produce a smaller model size but greatly increases conversion time. This setting only applies when options.preserve is false.
+ * @param {Boolean} [options.optimizeDrawCalls=false] Flag to merge duplicate vertices, which can produce a smaller model size but greatly increases conversion time. This setting only applies when options.preserve is false.
```

Comment:
Fix description.

Reply:
Good catch. Should be all good now @lilleyse